packages/gui/src/components/offers/OfferManager.tsx

```diff
@@ -171,13 +226,21 @@ function OfferList(props: OfferListProps) {
       {
         // eslint-disable-next-line react/no-unstable-nested-components -- The result is memoized. No performance issue
         field: (row: OfferTradeRecord) => {
-          const { createdAtTime } = row;
+          const { createdAtTime, status, validTimes } = row;
+          let countdownDisplay = null;
+          if (status !== 'CANCELLED' && validTimes.maxTime) {
+            countdownDisplay =
+              !isGetHeightInfoLoading && !isGetTimestampForHeightLoading && isWalletSynced && currentTime !== -20
```

Comment:
Can we figure out a better check than `currentTime !== -20`? It isn't obvious what this is for.